mcbackend/adapters/pymc.py

```diff
@@ -142,7 +142,9 @@ def setup(
                     name,
                     str(self.var_dtypes[name]),
                     list(self.var_shapes[name]),
-                    dims=list(self.model.RV_dims[name]) if name in self.model.RV_dims else [],
+                    dims=list(self.model.named_vars_to_dims[name])
+                    if name in self.model.named_vars_to_dims
```

Comment:
Seems a little unorthodox to use a ternary operator to compute the default arg. Couldn't it be just None and if it is None then it is computed? Just a nitpick though.

Reply:
I think I did this because it's in a list comprehension, and there's no way to conditionally NOT pass the kwarg